### PR TITLE
Focal support

### PR DESCRIPTION
**Pull request by chapulina**
The `focal` branch was branched off #17 .

### CODE REVIEW

**Comment by chapulina** [comment]
@j-rivero , gzdev seems to be having some trouble on Focal, see the results from using this branch on https://github.com/ignitionrobotics/ign-math/pull/139:

```
2020-07-29T21:18:53.8458438Z + /tmp/gzdev/gzdev.py repository enable --project=ignition-math6
2020-07-29T21:18:53.9206193Z Traceback (most recent call last):
2020-07-29T21:18:53.9209613Z   File "/tmp/gzdev/plugins/repository.py", line 84, in get_linux_distro
2020-07-29T21:18:53.9211127Z     distro_str = distro.linux_distribution()[0]
2020-07-29T21:18:53.9211930Z NameError: name 'distro' is not defined
2020-07-29T21:18:53.9212021Z 
2020-07-29T21:18:53.9212192Z During handling of the above exception, another exception occurred:
2020-07-29T21:18:53.9212308Z 
2020-07-29T21:18:53.9212468Z Traceback (most recent call last):
2020-07-29T21:18:53.9212628Z   File "/tmp/gzdev/gzdev.py", line 35, in <module>
2020-07-29T21:18:53.9212797Z     plugin.main()
2020-07-29T21:18:53.9212975Z   File "/tmp/gzdev/plugins/repository.py", line 192, in main
2020-07-29T21:18:53.9213151Z     process_input(args, config)
2020-07-29T21:18:53.9213355Z   File "/tmp/gzdev/plugins/repository.py", line 181, in process_input
2020-07-29T21:18:53.9213544Z     install_repos(project_list, config, linux_distro)
2020-07-29T21:18:53.9213723Z   File "/tmp/gzdev/plugins/repository.py", line 125, in install_repos
2020-07-29T21:18:53.9214152Z     install_repo(p['name'], p['type'], config, linux_distro)
2020-07-29T21:18:53.9214341Z   File "/tmp/gzdev/plugins/repository.py", line 128, in install_repo
2020-07-29T21:18:53.9214525Z     url = get_repo_url(repo_name, repo_type, config)
2020-07-29T21:18:53.9214709Z   File "/tmp/gzdev/plugins/repository.py", line 104, in get_repo_url
2020-07-29T21:18:53.9215145Z     if p['name'] == repo_name and p['linux_distro'].lower() == get_linux_distro():
2020-07-29T21:18:53.9215346Z   File "/tmp/gzdev/plugins/repository.py", line 86, in get_linux_distro
2020-07-29T21:18:53.9215535Z     distro_str = platform.linux_distribution()[0]
2020-07-29T21:18:53.9215941Z AttributeError: module 'platform' has no attribute 'linux_distribution'
```

File: entrypoint.sh [hunk 1]
```diff
@@ -7,6 +7,8 @@ OLD_APT_DEPENDENCIES=$1
 CODECOV_TOKEN=$2
 CMAKE_ARGS=$3
 
+export DEBIAN_FRONTEND="noninteractive"
```
Comment (mjcarroll) [inline entrypoint.sh:10]: Does this need to be backported into the bionic branch?

**Comment by j-rivero** [comment]
> @j-rivero , gzdev seems to be having some trouble on Focal, see the results from using this branch on [ignitionrobotics/ign-math#139](https://github.com/ignitionrobotics/ign-math/pull/139):

Ouch, this should fix it https://github.com/osrf/gzdev/pull/15

File: entrypoint.sh (hunk 1 above)
Comment (chapulina) [reply entrypoint.sh:10]: Even though it's not causing errors there, I think it's best to. I'll do it once this PR is in.

**Comment by chapulina** [comment]
`ign-math` is happy, so this first iteration is gtg

**Pull request merged**